libsignal-service/src/messagepipe.rs

```diff
@@ -138,6 +138,7 @@ impl MessagePipe {
 /// WebSocketService that panics on every request, mainly for example code.
 pub struct PanicingWebSocketService;
 
+#[allow(clippy::diverging_sub_expression)]
```

Comment:
Here too, a comment explaining why it triggers, and why we suppress would be nice.